cloud_functions/strava_auth_initiate/main.py

```
@@ -1,167 +1,175 @@
+                status=401,
+                headers=CORS_HEADERS,
+                response=json.dumps(
+                    {"error": "Unauthorized", "message": "Authorization code required"},
+                ),
+            )
+
+        # Retrieve secret
+        try:
+            strava_keys = get_secret(os.environ["STRAVA_SECRET"])
+            encryption_key = get_secret(os.environ["ENCRYPTION_SECRET"])["token"]
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)